pkg/component/gardenerscheduler/gardener_scheduler_test.go

```diff
@@ -88,7 +89,9 @@ var _ = Describe("GardenerScheduler", func() {
 
 		fakeClient = fakeclient.NewClientBuilder().WithScheme(operatorclient.RuntimeScheme).Build()
 		fakeSecretManager = fakesecretsmanager.New(fakeClient, namespace)
-		values = Values{}
+		values = Values{
+			RuntimeVersion: semver.MustParse("1.27.1"),
```

Comment:
Why do we set the version here and overwrite it in the `BeforeEach(...)` below?

Reply:
We only overwrite it for `< 1.26` case. For the other case, this value is used.

Reply:
My point was that we set it here and overwrite it in the common `BeforeEach(...)` in line 303. Then we overwrite it again in the `< 1.26` case in line 404.
I was just wondering if the tests are fine without this change as it seems to be overwritten anyway.
(It might only be relevant to the unchanged contexts `secrets`, `#Destroy` and `waiting functions`.)

Reply:
Yes, for `secrets`, it's needed.

Reply:
https://github.com/gardener/gardener/pull/8969/commits/06c792a3c8af4ecb6a0352cb60023075f7c9519a should do the trick.